config/ecs/config.go

```diff
@@ -32,6 +32,14 @@ func Configure(p *config.Provider) {
 	p.AddResourceConfigurator("aws_ecs_cluster", func(r *config.Resource) {
 		r.Version = common.VersionV1Alpha2
 		r.ExternalName = config.NameAsIdentifier
+		r.ExternalName.GetExternalNameFn = func(tfstate map[string]interface{}) (string, error) {
+			// expected id format: arn:aws:ecs:us-west-2:123456789123:cluster/example-cluster
+			w := strings.Split(tfstate["id"].(string), "/")
```

Comment:
should we check if `tfstate["id"]` is actually a string? e.g. would this line cause a crash if tfstate["id"] is nil?

Reply:
you can ignore this as others seem to be configured the same way and working fine.